packages/react-sdk/src/components/image-dev.stories.tsx

```diff
@@ -61,12 +60,29 @@ const imageLoader = USE_CLOUDFLARE_IMAGE_TRANSFORM
   ? cloudflareImageLoader
   : localImageLoader;
 
-const ImageTemplate: ComponentStory<
+const ImageBase: ComponentStory<
   React.ForwardRefExoticComponent<
-    ImageProps & { style: HTMLAttributes<"img">["style"] }
+    ImageProps & { style?: HTMLAttributes<"img">["style"] }
   >
 > = (args) => {
-  const style = { maxWidth: "100%", ...args.style };
+  warnOnce(
+    !(
+      imageMeta.defaultStyle?.maxWidth?.value === 100 &&
+      imageMeta.defaultStyle?.maxWidth?.unit === "%"
+    ),
+    "We expect that the image default style has maxWidth: 100%"
```

Comment:
I wonder where we could do this validation so that it can check also user-defined meta if user provides own components, probably in the utility function that is used by the component